boot/bootdata/hivedef.inf

```diff
@@ -2716,6 +3108,34 @@ TEMP_DIR="%USERPROFILE%\Lokale innstillinger\Midlertidig"
 
 ; LANG_POLISH SUBLANG_NEUTRAL
 [Strings.0015]
+EVENT_LABEL_0 = "Default Beep"
+EVENT_LABEL_1 = "Program error"
+EVENT_LABEL_2 = "Close program"
+EVENT_LABEL_3 = "Critical Battery Alarm"
+EVENT_LABEL_4 =  "Device Connect"
+EVENT_LABEL_5 = "Device Disconnect"
+EVENT_LABEL_6 = "Device Failed to Connect"
+EVENT_LABEL_7 = "Empty Recycle Bin"
+EVENT_LABEL_8 = "Low Battery Alarm"
+EVENT_LABEL_9 = "Maximize"
+EVENT_LABEL_10 = "Menu command"
+EVENT_LABEL_11 = "Menu popup"
+EVENT_LABEL_12 = "Minimize"
+EVENT_LABEL_13 = "New Mail Notification"
+EVENT_LABEL_14 = "Start Navigation"
+EVENT_LABEL_15 = "Open program"
+EVENT_LABEL_16 = "Print Complete"
+EVENT_LABEL_17 = "Restore Down"
+EVENT_LABEL_18 = "Restore Up"
+EVENT_LABEL_19 = "Asterisk"
+EVENT_LABEL_20 = "Exclamation"
+EVENT_LABEL_21 = "Exit ReactOS"
+EVENT_LABEL_22 = "Critical Stop"
+EVENT_LABEL_23 = "System Notification"
+EVENT_LABEL_24 = "Question"
+EVENT_LABEL_25 = "Start ReactOS"
+EVENT_LABEL_26 = "ReactOS Logoff"
+EVENT_LABEL_27 = "ReactOS Logon"
```

Comment:
for POLISH
EVENT_LABEL_0 = "Domyślny sygnał dźwiękowy"
EVENT_LABEL_1 = "Błąd programu"
EVENT_LABEL_2 = "Zamknięcie program"
EVENT_LABEL_3 = "Alarm o krytycznym poziome energii baterii"
EVENT_LABEL_4 =  "Podłączenie urządzenia"
EVENT_LABEL_5 = "Odłączenie urządzenia"
EVENT_LABEL_6 = "Niepowodzenie podłączenia urządzenia"
EVENT_LABEL_7 = "Opróżnienie kosza"
EVENT_LABEL_8 = "Alarm o niskim poziome energii baterii"
EVENT_LABEL_9 = "Maksymalizacja"
EVENT_LABEL_10 = "Polecenie menu"
EVENT_LABEL_11 = "Menu podręczne"
EVENT_LABEL_12 = "Minimalizacja"
EVENT_LABEL_13 = "Powiadomienie o nowej poczcie"
EVENT_LABEL_14 = "Rozpoczęcie nawigacji"
EVENT_LABEL_15 = "Otwarcie program"
EVENT_LABEL_16 = "Drukowanie ukończone"
EVENT_LABEL_17 = "Przywrócenie w dół"
EVENT_LABEL_18 = "Przywrócenie w górę"
EVENT_LABEL_19 = "Gwiazdka"
EVENT_LABEL_20 = "Wykrzyknik"
EVENT_LABEL_21 = "Zamknij system ReactOS"
EVENT_LABEL_22 = "Zatrzymanie krytyczne"
EVENT_LABEL_23 = "Powiadomienie systemowe"
EVENT_LABEL_24 = "Pytanie"
EVENT_LABEL_25 = "Uruchomienie systemu ReactOS"
EVENT_LABEL_26 = "Wylogowywanie z systemu ReactOS"
EVENT_LABEL_27 = "Logowanie do systemu ReactOS"

Reply:
Please don't bikeshed on people's pull requests. Make a PR in your own with the translated strings. Thanks for your understanding.

Reply:
It's not bikehed, both are mine and both are covered by this single PR
Furthermore, I needed to provide the FR-FR exemple to show that the localization mechanism work. Some other PR have been put on hold because of the lack of such evidence that the proposed design fully work.
On top, I needed to provide exemple to another contributor who is willing to add pt-pt in a dedicated PR as per discussion on the dev channel.

Therefore, I propose to keep full scope and to have a review with a second FR native reviewer (I'm personally FR native speaker)

Reply:
In case you didn't understand, it was directed at @pithwz and not you.

Reply:
My mistake.. sorry.
I fully support your view @Fraizeraust !
@pithwz  : Please wait for merge then open your dedicated PR.